.gitignore

```diff
@@ -1 +1,5 @@
 .DS_Store
+src/pg/observatory--current--dev.sql
+src/pg/observatory--dev--current.sql
+src/pg/observatory--dev.sql
+src/pg/test/results
```

Comment:
Being picky, here, but that would ignore a file named results too, to ignore only a results directory it would be `src/pg/test/results/`